modules/apps/frontend-theme/frontend-theme-dialect/src/css/dialect/variables/_navs.scss

```diff
@@ -0,0 +1,413 @@
+			height: var(--tab-tertiary-item-active-bar-thickness),
+			width: var(--tab-tertiary-horizontal-item-active-bar-width),
+		),
+	),
+	disabled: (
+		background-color: var(--tab-tertiary-item-background-color-disabled),
+		color: var(--tab-tertiary-item-color-disabled),
+		font-weight: var(--tab-tertiary-item-font-weight),
+		after: (
+			content: none,
+		),
+		inline-item: (
+			color: var(--tab-tertiary-item-icon-color-disabled),
+		),
+	),
+	inline-item: (
+		color: var(--tab-tertiary-item-icon-color),
+		font-size: var(--tab-tertiary-item-icon-font-size),
+		margin-y: var(--tab-tertiary-item-inline-item-spacer-y),
+	),
+	inline-item-after: (
+		margin-left: var(--tab-tertiary-item-inline-item-spacer-x),
+	),
+	inline-item-before: (
+		margin-right: var(--tab-tertiary-item-inline-item-spacer-x),
+	),
+);
+
+$nav-tertiary-link-vertical: (
+	hover: (
+		after: (
+			background-color: var(--tab-tertiary-item-active-bar-color-active),
+			border-radius: var(--tab-tertiary-item-active-bar-border-radius),
+			height: var(--tab-tertiary-vertical-item-active-bar-height),
+			top: 0,
+			width: var(--tab-tertiary-item-active-bar-thickness),
+		),
+	),
+	active-class: (
+		border-width: 0,
+		after: (
+			background-color: var(--tab-tertiary-item-active-bar-color-active),
+			border-radius: var(--tab-tertiary-item-active-bar-border-radius),
+			height: var(--tab-tertiary-vertical-item-active-bar-height),
+			top: 0,
+			width: var(--tab-tertiary-item-active-bar-thickness),
+		),
+	),
+);
+
+$nav-tertiary-vertical: (
+	flex-direction: column,
+	width: fit-content,
+);
+
+$nav-tertiary-active-bar-top: ();
+
+$nav-tertiary-link-active-bar-top: (
+	border-width: 0.063rem 0 0,
+	hover: (
+		after: (
+			top: 0,
+		),
+	),
+	active-class: (
+		after: (
+			top: 0,
+		),
+	),
+);
+
+// Segment tab
+
+$nav-segment: (
+	background-color: var(--tab-segment-background-color),
+	border-color: var(--tab-segment-border-color),
+	border-radius: var(--tab-segment-border-radius),
+	border-width: var(--tab-segment-border-width),
+	border-style: solid,
+	padding: var(--tab-segment-padding-x) var(--tab-segment-padding-y),
+	width: fit-content,
+);
+
+$nav-segment-link: (
+	background-color: var(--tab-segment-item-background-color),
+	border-color: var(--tab-segment-item-border-color),
+	border-radius: var(--tab-segment-item-border-radius),
+	border-width: var(--tab-segment-item-border-width),
+	box-shadow: none,
+	color: var(--tab-segment-item-color),
+	font-size: var(--tab-segment-item-font-size),
+	font-weight: var(--tab-segment-item-font-weight),
+	line-height: var(--tab-segment-item-line-height),
+	after: (
+		content: none,
+	),
+	hover: (
+		background-color: var(--tab-segment-item-background-color-hover),
+		border-color: var(--tab-segment-item-border-color-hover),
+		box-shadow: var(--tab-segment-item-box-shadow-hover),
+		color: var(--tab-segment-item-color-hover),
+		after: (
+			content: none,
+		),
+	),
+	active-class: (
```

Comment:
Icon color needs to be set when active and hovered.

Reply:
Fixed